app/src/main/java/com/yacgroup/yacguide/database/tourbook/TourbookExporter.kt

```diff
@@ -41,11 +49,57 @@ class TourbookExporter(
     private val _partnersKey = "partners"
     private val _notesKey = "notes"
 
+    enum class ExportFormat {
+        eJSON,
+        eJSONVEROBSE,
+        eCSV;
+    }
+
+    var exportFormat: ExportFormat = ExportFormat.eJSON
```

Comment:
Dangerous without unit tests. Public state variables are pretty error-prone. This info should actually be passed as local parameter to `exportTourbook()`

Reply:
Why that? It is a enum. Hence, it can only set to dedicated values?

Reply:
Original I wanted to pass this as local parameter but the problem is that the format is set in a different method than the actual export is done. Hence, I would need a variable to store this information anyway. So, I decided to store it directly in the object. The other approach would be to pass it through all methods, which is in my eyes very nasty. Or is there a smarter way to do this?

Reply:
> Why that? It is a enum. Hence, it can only set to dedicated values?

It's not about the type but the fact that you can manipulate a state variable from outside. This may result in tricky bugs. You can just leave it as is, maybe I'll have a look at it again when doing the refactoring issue for next release.